doc/wallet-passphrase.md

```diff
@@ -0,0 +1,21 @@
+## Wallet Passphrase
+
+Understanding the nuances of wallet security is essential to storing your Bitcoin safely. One crucial aspect of safeguarding funds is the utilization of a wallet passphrase or password, implemented through wallet encryption. Let's explore key points to demystify the nature of a wallet passphrase and the encryption process, emphasizing what they do and what they don't do:
+
+  - **Not the Seed:**
```

Comment:
I think this part can be shortened a lot.

Reply:
fixed